jobmon_server/src/jobmon/server/web/error_log_clustering.py

```diff
@@ -1,73 +1,42 @@
 import nltk
-from nltk import corpus, sent_tokenize, word_tokenize
-from nltk.stem.porter import PorterStemmer
+from nltk import corpus
 from pandas import DataFrame
 import sklearn.feature_extraction.text as ext
 
+nltk.download("punkt", quiet=True)
+nltk.download("stopwords", quiet=True)
+
+stopwords = corpus.stopwords.words("english")
+stopwords.extend(["could", "might", "must", "need", "sha", "wo", "would"])
+
 
 def cluster_error_logs(df: DataFrame) -> DataFrame:
     """Cluster error logs using unsupervised learning."""
+    count_vectorizer = ext.CountVectorizer()
+    doc_matrix = count_vectorizer.fit_transform(df["error"])
```

Comment:
This is the wrong column. We need to be clustering on task_instance_stderr_log. Otherwise looks great

Reply:
Yep, was actually just looking at this since we got a ticket about it. Prod is currently clustering on the wrong thing. Just updated it.